Wrote Configurator VTX Tab Wiki (#469)
betaflight/betaflight.com: docs/wiki/configurator/vtx-tab.md

# Video Transmitter Tab

> Set up your video transmitter using this tab. It allows you to set the power, band, and channels for your VTX.

Reviewer: ```suggestion
Configure the video transmitter (VTX) using this tab. It allows setting of the power, band, channels, etc.
```

:::note

> If you're using a DJI or Walksnail video transmitters, you can ignore this tab.

Reviewer 2: ```suggestion
When using a DJI or Walksnail video transmitters, this tab can be ignored.
```

This is only used for analog and HDZero video transmitters.
:::

![VTX tab](/img/betaflight_configurator_vtx_tab.png)

It is important to have VTX Telemetry (SmartAudio or Tramp) enabled in the Ports tab to use this feature.

## Selected Mode

| **Option**                   | **Description**                                                                                                                                                                                                                                                                                                                                                                     |
| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | **Enter frequency directly** | If you enable this, the Configurator will let you select a frequency in place of the habitual band/channel. For this to work your VTX must support this feature.                                                                                                                                                                                                                    |

Reviewer 2: ```suggestion
| **Enter frequency directly** | When enabled, the Configurator will let you select a frequency in place of the habitual band/channel. For this to work your VTX must support this feature.                                                                                                                                                                                                                    |
```
